services/user-management/src/main/java/com/foodie/user/controllers/PermissionController.java

```diff
@@ -78,6 +81,13 @@ public ResponseEntity<FoodieBaseResponse> deletePermission(@Valid @PathVariable(
 
         log.info("Deleting permission with ID: {}", id);
 
+        // Check if the permission exists
+        Optional<Permission> permission = permissionService.findByIdAndDeletedAtIsNull(id);
+
+        if (permission.isEmpty()) {
+            return ResponseEntity.badRequest().body(new FoodieBaseResponse("Permission not found"));
```

Comment:
The style here is very different from line `34` you should try and be consistent in your styles

Reply:
notfound does not allow you to set the body